### ISSUE TITLE
ngrx 19 support

### ISSUE THREAD

**Issue by baffioso** [angular-architects/ngrx-toolkit#107]
ngrx is getting close of releasing version 19 with a release candidate. Any plans on creating RC for ngrx-toolkit or are you waiting for final release?

**Comment by rainerhahnekamp** [comment]
Hi @baffioso, yes, I'd like to wait for the RC. The plan is that the ngrx-toolkit gets an enhanced devtools support, and maybe an additional feature for resetting states.

**Comment by the-ult** [comment]
Any news on the ngrx-toolkit upgrade?

There seems to be a problem with `withCallState()`

```ts
withCallState(),
withComputed(({  loaded, error }) => ({})
```

> Property 'error' does not exist on type..

Or is there a proper way we should fix/migrate this now?

**Comment by rainerhahnekamp** [comment]
Yes, looks like it is going to be a Christmas present. @the-ult that error happens in SignalStore 19 or also in v18?

**Comment by rainerhahnekamp** [comment]
Thanks @the-ult, yeah, that's probably because of the changes to `withProps`.

Support for `rxResource` is a topic on its own. We'd like to wait for the `httpResource` and then we'll see how it can be integrated into the SignalStore. At the moment, you need to do it via `withProps`. There is an example in the blog post btw: https://dev.to/ngrx/announcing-ngrx-19-ngrx-signals-features-action-signals-and-more-2b35

**Comment by BDominik** [comment]
I had trouble to upgrade my monorepo to angular 19 because of the toolkit. I had issues with npm i.
So I removed it, and did the migration.
But right now I cannot even install the toolkit.

![image](https://github.com/user-attachments/assets/43afc7e2-49e0-47d7-aedb-50c616fbbf07)

**Comment by thetric** [comment]
#114 is merged, so this can be closed?

**Comment by rainerhahnekamp** [comment]
Yes, we can close that one. The devtools have been rewritten from scratch and come now with additional features. That's was the reason for the delay. The rest was just easy peasy.
